package.json

```diff
@@ -43,33 +43,33 @@
     "pull-stream": "^3.6.14",
     "push-stream": "^11.0.0",
     "rimraf": "^3.0.2",
-    "ssb-about-self": "^1.0.1",
+    "ssb-about-self": "^1.1.0",
     "ssb-box": "^1.0.0",
-    "ssb-box2": "^2.0.2",
+    "ssb-box2": "^3.0.0",
```

Comment:
only *major* update

Reply:
I highly suspect the tests are breaking because of one of these deps. The easy way to fix the tests is to revert these changes and bump *only* ssb-box2. Unless you have the patience to debug which of these deps broke the tests and why...